specs/windows/interface_details.table

```diff
@@ -0,0 +1,33 @@
+table_name("interface_details")
```

Comment:
I know, but forcing a single schema across platforms for each table "concept" was one of the original goals. That has created a lot of assumptions within build/tooling that results in not supporting same-name-different schema.

For a better user experience we could consider "hiding" several columns per-platform. So the columns/types are still shared, but the default result of `SELECT *` is altered?